packages/contracts-communication/contracts/InterchainApp.sol

```diff
@@ -90,8 +92,9 @@ contract InterchainApp {
     event AppMessageSent();
 
     function send(bytes32 receiver, uint256 dstChainId, bytes calldata message) external payable {
+        bytes memory options = OptionsLib.encodeOptionsV1(OptionsV1(200_000, 0));
```

Comment:
Small nit:  `OptionsV1(200_000, 0).encodeOptionsV1()` will also work (even without "using OptionsLib" statement)

Reply:
resolved in #2078